module-vfs/src/purefs/fs/notifier.cpp

```diff
@@ -109,7 +109,7 @@ namespace purefs::fs::internal
             svc->bus.sendUnicast(std::move(msg), svc->GetName());
         }
         else {
-            LOG_WARN("Sent notification to the same thread is forbidded");
+            LOG_WARN("Sent notification to the same thread is forbidden");
```

Comment:
If you're changing this, I think the correct grammar should be "Sending a notification to the same thread is forbidden."